examples/shaders/shared/src/lib.rs

```diff
@@ -13,9 +19,9 @@ use spirv_std::glam::{vec3, Vec3};
 #[cfg(target_arch = "spirv")]
 use spirv_std::num_traits::Float;
 
+#[spirv(block)]
 #[derive(Copy, Clone)]
 #[allow(unused_attributes)]
-#[spirv(block)]
```

Comment:
any reason for this, or just formatting?

Reply:
You can't put proc macros bellow derives.